.github/workflows/ci.yml

```diff
@@ -25,6 +25,8 @@ jobs:
         uses: "actions/checkout@v4"
       - name: "Setup environment"
         uses: "networktocode/gh-action-setup-poetry-environment@v6"
+        with:
+          poetry-version: "1.8.2"
```

Comment:
Any particular reason for 1.8.2 rather than 1.8.5?

Reply:
The lock file for this and other projects is currently 1.8.2, but I can update if you would prefer me use 1.8.5.

Reply:
Sounds like as good a reason as any :)